pgdb/generate.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# Usage
+# /generate.sh database start end [agenda-votes]
+
+# Generates a .sql for a subset of the desired dcrdata `database`, with blocks in
+# the range `start` to `end`. An optional parameter, `agenda-votes`, will set
+# the number of random agenda votes selected. If not set, the number of votes
+# will be set to the number of blocks in the range.
+
+TEMPLATE=`cat template.sql`
```

Comment:
Seeing as we're using bash, this might as well be `TEMPLATE=$(<template.sql)` to avoid spawning cat.